Aliases/Server - Heroism & Shame/dt/adult.alias

```diff
@@ -14,6 +14,12 @@ me.create_cc_nx("DT Slot", "0", "2", "none", "bubble")
 perctup, invetup, perftup, perstup = ("perception", me.skills.perception.value), ("investigation", me.skills.perception.value), ("performance", me.skills.performance.value), ("persuasion", me.skills.persuasion.value)
 searchbonus, searchskill, seductbonus, seductskill, perfconbonus = invetup[1] if invetup[1] > perctup[1] else perctup[1], invetup[0] if invetup[1] > perctup[1] else perctup[0], perftup[1] if perftup[1] > perstup[1] else perstup[1], perftup[0] if perftup[1] > perstup[1] else perstup[0], (perftup[1] - charismaMod) + constitutionMod
 
+# checks for guidance and adds it
+if "guidance" in me.spellbook:
+    searchbonus = str(searchbonus)+"+1d4[guidance]"
+else:
+    pass
```

Comment:
Can just remove this since it does nothing and else blocks are not required in drac2

Reply:
fixed on new push